Core/include/Acts/Geometry/GeometryObject.hpp

```diff
@@ -35,21 +35,21 @@ class GeometryObject {
 
   /// Constructor from a value
   ///
-  /// @param geoID the geometry identifier of the object
-  GeometryObject(const GeometryID& geoID) : m_geoID(geoID) {}
+  /// @param geometryId the geometry identifier of the object
+  GeometryObject(const GeometryID& geometryId) : m_geometryId(geometryId) {}
```

Comment:
Did you do a global replacement? 

I replaced only the method calls ...

Reply:
yup